x-pack/plugins/observability/public/pages/rules/state_container/state_container.tsx

```diff
@@ -1,68 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import {
-  createStateContainer,
-  createStateContainerReactHelpers,
-} from '@kbn/kibana-utils-plugin/public';
-import { RuleStatus } from '@kbn/triggers-actions-ui-plugin/public';
-
-interface RulesPageContainerState {
-  lastResponse: string[];
-  status: RuleStatus[];
-}
-
-const defaultState: RulesPageContainerState = {
-  lastResponse: [],
-  status: [],
-};
-
-interface RulesPageStateTransitions {
-  setLastResponse: (
-    state: RulesPageContainerState
-  ) => (lastResponse: string[]) => RulesPageContainerState;
-  setStatus: (state: RulesPageContainerState) => (status: RuleStatus[]) => RulesPageContainerState;
-}
-
-const transitions: RulesPageStateTransitions = {
-  setLastResponse: (state) => (lastResponse) => {
```

Comment:
All this code seems to be referencing a previous implementation of this table that is not used in the current page.

I suspect the RuleList component used to live inside Observability, but has been moved to TriggersActionUi. In that process this implementation was not properly cleaned up.